### PR TITLE
Feature/book details test cag

### PR DESCRIPTION
**Pull request by cagallo**
## What Features are Being Added?

- Added class names to BookDetails.js
- Created test for single page book view- book_details_spec.js
- Created fixture for single page book view 

## What is Being Refactored?

- N/A

## Where should the reviewer start?

- book_details_spec.js

## How should this be manually tested?

- npm run cypress 
- click on book_details_spec.js

## What are the relevant tickets?

- Cypress testing 

## Any Other Notes/Questions/Problems?

- None! Happy we have some tests to show Robbie!

### CODE REVIEW

File: cypress/fixtures/singleBook.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[
```
Comment (krdurrett) [inline cypress/fixtures/singleBook.json:1]: These fixtures are a great idea!

Comment (Fordo29) [inline cypress/fixtures/singleBook.json:1]: Nice add to make it an array of object to match what is returned on the GET request

File: cypress/integration/book_details_spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+describe('Book Club book details page user flow', () => {
+
+  it('should display all book details for a single book', () => {
+    cy.intercept('https://book-club-api-2110.herokuapp.com/api/v1/books/2', { fixture: 'singleBook.json' })
```
Comment (krdurrett) [inline cypress/integration/book_details_spec.js:4]: nice full testing for book details

**Review by krdurrett — APPROVED** [review]
The test runs perfect!

File: src/components/BookDetails.js [hunk 3]
```diff
@@ -26,12 +26,12 @@ class BookDetails extends Component {
         <section className="book-details">
           <img src={this.state.selectedBook.book_image}/>
           <div className="book-details-text-container">
-            <h2>{this.state.selectedBook.title}</h2>
-            <p>{this.state.selectedBook.author}</p>
-            <p>{this.state.selectedBook.description}</p>
+            <h2 className="selected-title">{this.state.selectedBook.title}</h2>
```
Comment (Fordo29) [inline src/components/BookDetails.js:29]: Good call on adding classNames for the elements for easier css later

File: cypress/integration/book_details_spec.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+describe('Book Club book details page user flow', () => {
```
Comment (ectweitmann) [inline cypress/integration/book_details_spec.js:1]: Looks good, we'll just need to remember to come back and refactor so the tests stay up to date!

**Review by ectweitmann — APPROVED** [review]

**Pull request merged**